chal18.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <body>
+    <script type="text/javascript">
+      let numbers = [Math.random(),Math.random(),Math.random(),Math.random(),Math.random()];
+      console.log(numbers);
+      function sorter() {
+        for (var foo = 0; foo < numbers.length; foo++) {
```

Comment:
These should be `let` and I would name the variables better. I would go with `i`, `j` in this case.